src/main/webapp/index.html

```diff
@@ -3,17 +3,18 @@
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
-<script src="script.js"></script>
 <form action="/lecture" method="POST">
-  <h2>Add a new lecture!<h2>
+  <h2>Add a new lecture!</h2>
   <h3>Enter a name for this lecture:</h3>
-  <input type="text" placeholder="Lecture Name" name="name-input">
-  <br/>
-  <h3>Enter the YouTube link for this lecture:<h3>
-  <input type="text" placeholder="Name" name="video-input">
-  <br/>
-  <br/>
+  <input type="text" placeholder="Lecture Name" name="name-input" />
```

Comment:
You shouldn't have a backslash for br and input 
Source: https://google.github.io/styleguide/htmlcssguide.html#Document_Type

Reply:
`make pretty` adds those